File: Easy/53- 171. Excel Sheet Column Number/171. Excel Sheet Column Number.cs
public class Solution {
        public int TitleToNumber(string columnTitle)
        {
            int ans = 0;
            for(int i =0 ; i < columnTitle.Length; i++)
            {
                ans *= 26;
                ans += columnTitle[i] - '@';
            }
            return ans;
        }
}

File: Easy/53- 171. Excel Sheet Column Number/171. Excel Sheet Column Number.js
/**
 * @param {string} columnTitle
 * @return {number}
 */
var titleToNumber = function (columnTitle) {
    const charCodeBase = "@".charCodeAt(0);
    let ans = 0;
    for (var i = 0; i < columnTitle.length; i++) {
        ans *= 26;
        ans += columnTitle.charCodeAt(i) - charCodeBase;
    }
    return ans;
};
